cmd/run.go

```diff
@@ -90,6 +90,10 @@ Non-bundle data file and directory paths can be prefixed with the desired
 destination in the data document with the following syntax:
 
 	<dotted-path>:<file-path>
+		
+To set a data file as the input document in the REPL prefix the file path:
```

Comment:
nit: Please switch to "interactive shell" rather than "REPL" to keep it consistent with the rest of the docstring.

Reply:
@patrick-east , thanks for you precious reviews , I fixed that at commit 3945a1d.